src/app/modules/footer/FooterItems.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+function FooterItems({heading,  items, path}) {
+    var menuItems = items.map((item)=>{
+        return <li className="footer__link"><a href={path}>{item.name}</a></li>
+    })
+    return (
+            <div className="col-sm-12 col-md-2">
+                <h4 className="footer__title"><a href="/">{heading}</a></h4>
+                <ul className="list-unstyled">
+                    {menuItems}
+                </ul>
+            </div>
+    )
+}
+export default FooterItems;
```

Comment:
Chèn ký tự xuống dòng vào cuối file.

Lý do: https://int3ractive.com/2019/04/why-trailing-comma-in-js-is-a-good-idea.html
Cài extension editorconfig để nó tự động thêm.